Source/Libraries/openXDA.Adapters/PQMarkController.cs

```diff
@@ -479,7 +488,7 @@ public IHttpActionResult CreateRecord(string modelName, [FromBody] JObject recor
 
                     PropertyInfo assetKeyProperty = type.GetProperty("AssetKey");
 
-                    if (!(assetKeyProperty is null))
+                    if (!(assetKeyProperty is null)) //TODO: Make sure this works with asset subtypes
```

Comment:
Do we still need to verify this?

Reply:
It's verified, I simply forgot to delete this comment.